src/data-sources/plugins/DataSourcePlugin.ts

```diff
@@ -31,7 +31,7 @@ export interface IDataSourcePlugin {
   };
 
   bind (actionClass: any): void;
-  updateDependencies (dependencies: IDictionary, args: IDictionary, callback: () => void): void;
+  _updateDependencies (dependencies: IDictionary, args: IDictionary, callback: () => void): void;
```

Comment:
This was done to indicate a private wrapper. I've renamed to make this more explicit in c4c5c0e.